src/v2/spec.yaml

```diff
@@ -2176,6 +2176,31 @@ components:
                 $ref: "#/components/schemas/CastWithInteractions"
         next:
           $ref: "#/components/schemas/NextCursor"
+    ConversationSummary:
+      type: object
+      required:
+        - summary
```

Comment:
Should this be called `conversation` cuz there is no `summary` property in this object